src/lib/components/ratio/index.svelte

```diff
@@ -19,6 +41,15 @@
       <span class="sr-only">The contrast ratio is</span>
```

Comment:
I know it's not new here, but I don't think this is useful screen reader context. The heading directly before this says 'current ratio' already.

Reply:
Agreed. On the topic, I wonder if (in another story) we should edit the click-to-copy buttons to say what exactly they are copying?

Reply:
Yeah, I don't know the best solution on those - but no harm oping an issue to explore it.